R/pkg/R/serialize.R

```diff
@@ -71,18 +69,19 @@ writeObject <- function(con, object, writeType = TRUE) {
   switch(serdeType,
          NULL = writeVoid(con),
          integer = writeInt(con, object),
+         glue =,
          character = writeString(con, object),
          logical = writeBoolean(con, object),
-         double = writeDouble(con, object),
+         double =,
```

Comment:
R does cascading `switch` (like C), so e.g. `'double'` will "fall down the ladder" til the next non-empty entry, namely, `numeric`